astropy/io/votable/tests/table_test.py

```diff
@@ -75,6 +75,9 @@ def test_read_through_table_interface(tmpdir):
 
     assert len(t) == 5
 
+    # Issue 8354
+    assert t['float'].format is None
```

Comment:
I am not convinced that this sufficiently tests this patch. Can you please elaborate?

How about turning your original example in #8354 (the one that does not use `astroquery`) into a test?

Reply:
As advocated by @taldcroft in [this comment](https://github.com/astropy/astropy/issues/8354#issuecomment-456049191), Table outputs full-precision values when `format` is `None`. The original motivation to use `repr` in the format, when converting from VOTable to Table, was to preserve precision. Therefore, the upstream fix is to ensure that floating-point columns have their `.format` attribute set to `None`.

The problem my IPAC colleagues reported came from using astroquery. In #8354 I made a contrived, shortcut example in which I set `.format` to a `repr`-style value. It is highly unlikely that a user will do this. I think I should edit the problem description in #8354 to reflect that.